src/LaunchDarkly.Client/ILoggerExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;

```

Comment:
0 is the event ID.  there's already overloads for passing the exception along, but because the event id is first, I imagine most people don't use this API correctly.  :(